src/components/PageElements/Elements/ReviewApplicantResponse.tsx

```diff
@@ -10,24 +10,40 @@ interface ReviewApplicantResponseProps {
   applicationResponse: ApplicationResponse
   summaryViewProps: SummaryViewWrapperProps
   reviewResponse?: ReviewResponse
+  previousReviewResponse?: ReviewResponse
   isActiveReviewResponse: boolean
   isNewApplicationResponse: boolean
+  isNewReviewResponse?: boolean
+  isChangeRequest?: boolean
   showModal: () => void
 }
 
 const ReviewApplicantResponse: React.FC<ReviewApplicantResponseProps> = ({
   applicationResponse,
   summaryViewProps,
   reviewResponse,
+  previousReviewResponse,
   isNewApplicationResponse,
   isActiveReviewResponse,
+  isNewReviewResponse = false,
```

Comment:
I think we should show 'updated' whenever response is updated (when isChanged === true), currently isChanged would only be set during change request phase (so this should work correctly and shouldn't interfere with re-reviews).

It looks like isNewReviewResponse =  isChangeRequest && isChanged, i think it should just be = isChanged.

Reply:
Good catch : ) 6c35a54